examples/graphupscale.cpp

```diff
@@ -165,9 +165,11 @@ int main(int argc, char* argv[])
 
 #if SMOOTHG_USE_SAAMGE
         SAAMGeParam saamge_param;
+        const bool coarse_coefficient = false;
         const auto hbsa_upscale = GraphUpscale(comm, vertex_edge, coarse_factor,
                                                spect_tol, max_evects, dual_target,
                                                scaled_dual, energy_dual, use_hybridization,
+                                               coarse_coefficient,
```

Comment:
I added this argument so the code will compile with SAAMGe enabled. It doesn't really belong in this PR, let me know if it's an issue and I'll remove it and issue a separate PR.

Reply:
Its fine here :+1: